File: sib_backend/SIB.Domain/Entities/EntidadGubernamental.cs
using System;

namespace SB.Management.Domain.Entities
{
    public class EntidadGubernamental
    {
        public int Id { get; set; }
        public string Nombre { get; set; } = string.Empty;
        public string Categoria { get; set; } = string.Empty;
        public string PoderDelEstado { get; set; } = string.Empty;
        public string Sector { get; set; } = string.Empty;
    }
}

File: sib_frontend/src/types/entidadGubernamental.ts
export type EntidadGubernamental = {
  id: number
  nombre: string
  categoria: string
  poderDelEstado: string
  sector: string
}

export type CrearEntidadGubernamentalPayload = {
  nombre: string
  categoria: string
  poderDelEstado: string
  sector: string
}